source/documentation/ingestion/cadet-registration.md

```diff
@@ -1,24 +1,64 @@
 # Adding tables to Find MoJ data from Create a Derived Table (CaDeT)
 
-- Find MoJ data is integrated with the [Create a Derived Table](https://github.com/moj-analytical-services/create-a-derived-table) service (CaDeT).
-- DBT models created from CaDeT are represented by tables in Find MoJ data.
-- To add a model created in CaDeT to Find MoJ data, add the tag `dc_display_in_catalogue` to that model. [Config of CaDeT models is described in their documentation here.](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/create-a-derived-table/models/#where-can-i-define-configs)
-- To add an owner to a table in Find MoJ data, add the `dc_owner` metadata item to the CaDeT model. This represents the data owner of the data held within the model. The owner should correspond to an existing @justice.gov.uk email address. For example, `john.smith@justice.gov.uk` would use `john.smith` as the entry for `dc_owner`.
-- Other metadata fields which can be added to CaDeT models to be displayed in Find MoJ data are:
-  - `dc_slack_channel_name`: slack channel name to contact about the data
-  - `dc_slack_channel_url`: slack channel url to contact about the data
-  - `dc_where_to_access_dataset`
-- Domain and database for a DBT model in Find MoJ data are inferred from the CaDeT file structure.
-
-For example, in the `dbt_project.yml` file:
+Find MoJ data uses the [Create a Derived Table](https://github.com/moj-analytical-services/create-a-derived-table) service (CaDeT) as a source of metadata about the Analytical Platform. CaDeT uses a python package called [dbt](https://www.getdbt.com/product/what-is-dbt).
+
+By default, all models and sources will be ingested into the Datahub catalogue, but they will not be shown in the Find MoJ data service.
+
+## Make a model or source visible
```

Comment:
We may want to split this by model and source, since sources can't be added to `dbt_project.yml` - you need to edit the property file instead